### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site-Add Bindings_Keyboard: Keyboard Focus is not visible on some of the controls while navigating through the keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#135]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS2.4.7

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Navigate to "bindings" page tab.
7. Tab till "Add" button and Press Enter.
8. Verify that focus is visible on all the controls properly while navigating through the keyboard.

**Actual Result:**
Focus is not visible at the following controls while navigating through the keyboard:
"Save" and "Cancel" icon, Refresh Button, Checkboxes, Toggle Buttons, Select Certificate Button.

**Expected Result:**
When any interactive control receives keyboard focus, a rectangular border should appear around these controls so that user can differentiate clearly between the focused and non-focused controls on the page. 

**User Impact:**
This Success Criterion helps anyone who relies on the keyboard to operate the page, by letting them visually determine the component on which keyboard operations will interact at any point in time.
People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
![bug 135_focus visibility issue-1](https://user-images.githubusercontent.com/38521133/44717424-92f62f00-aada-11e8-9326-0f13edb3ce1a.png)
![bug 135_focus visibility issue-2](https://user-images.githubusercontent.com/38521133/44717425-92f62f00-aada-11e8-91a0-550f1edffbc8.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
